### DISCUSSION TITLE
希望能够加入倍速功能和稍后再看功能

### DISCUSSION THREAD

**Discussion by Potaperper** [xfangfang/wiliwili#74] [Ideas]
我使用了一下软件，目前确实主要功能已经相当完善。
可惜倍速和稍后再看功能好像没有，这两个功能还是较为常用的，希望作者能够加入这两个功能，非常感谢。

**Comment by xfangfang** [comment]
@POTATOPEPPER2077 

https://github.com/xfangfang/wiliwili/discussions/47


稍后再看由他在写：
@Haifeng97
